dxr/plugins/python/tests/test_modules_packages/code/common_name_module.py

```diff
@@ -0,0 +1 @@
+"""Module with a common filename for testing multiple results."""
```

Comment:
Nitty nit nit: this isn't a sentence, so the gods do not grant it a period. (And I just used a colon wrong. ;-))